tools/bridge/requirements.txt

```diff
@@ -5,4 +5,5 @@ toml
 web3
 contract-deploy-tools
 python-dotenv
+validators
```

Comment:
Please also add this to setup.py

Reply:
You mean constraints.txt?

Reply:
No, I did mean the setup.py. But it doesn't declare any dependency, i.e. all of them are missing anyway. so, feel free to keep it as it is.

Reply:
Ok. Added to constraints.txt anyway.